src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkOptimizedIntegrationTest.java

```diff
@@ -91,18 +94,18 @@ private String getCloudInputs() {
     public Object[][] createBQSRTestDataBucket() {
         final String GRCh37Ref = ReferenceAPISource.URL_PREFIX + ReferenceAPISource.GRCH37_REF_ID;
         final String localResources =  getResourceDir();
-        final String HiSeqBamCloud = getCloudInputs() + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam";
-        final String dbSNPb37 =  getResourceDir() + "dbsnp_132.b37.excluding_sites_after_129.chr17_69k_70k.vcf";
+        final String HiSeqBamCloud_chr20 = getCloudInputs() + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam";
+        final String dbSNPb37_chr20 = localResources + "dbsnp_138.b37.excluding_sites_after_129.ch20.1m-1m1k.vcf";
 
         return new Object[][]{
                 // input in cloud, computation local.
-                {new BQSRTest(GRCh37Ref, HiSeqBamCloud, dbSNPb37, "", localResources + "expected.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.recal.txt")},
+                {new BQSRTest(GRCh37Ref, HiSeqBamCloud_chr20, dbSNPb37_chr20, "", localResources + "expected.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.recal.txt")},
         };
     }
 
 
     // "local", but we're still getting the reference from the cloud.
-    @Test(dataProvider = "BQSRTest", groups = {"cloud"})
+    @Test(dataProvider = "BQSRTest", groups = {"cloud"}, enabled = false)
```

Comment:
Please add a comment here explaining why this test is disabled, and file a new bug against `BaseRecalibratorSparkOptimized` explaining the problem and how to reproduce.